### ISSUE TITLE
New language: uz

### ISSUE THREAD

**Issue by malikovss** [squidfunk/mkdocs-material#3906]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translation

```jinja
{% macro t(key) %}{{ {
  "language": "uz",
  "direction": "ltr",
  "clipboard.copy": "Buferga nusxalash",
  "clipboard.copied": "Buferga nusxalandi",
  "edit.link.title": "Ushbu sahifani tahrirlash",
  "footer.previous": "Oldingi sahifa",
  "footer.next": "Keyingi sahifa",
  "footer.title": "Pastgi qism",
  "header.title": "Sarlavha",
  "meta.comments": "Izohlar",
  "meta.source": "Manba",
  "nav.title": "Navigatsiya",
  "search.placeholder": "Qidirish",
  "search.share": "Ulashish",
  "search.reset": "Tozalash",
  "search.result.initializer": "Qidiruv ishga tushirilmoqda",
  "search.result.placeholder": "Qidiruvni boshlash uchun kiriting",
  "search.result.none": "Mos natijalar yo'q",
  "search.result.one": "1 ta mos natija",
  "search.result.other": "# ta mos keladigan natijalar",
  "search.result.more.one": "Ushbu sahifada yana 1 ta natija",
  "search.result.more.other": "Bu sahifada yana # ta natija",
  "search.result.term.missing": "To'ldirilmagan",
  "search.title": "Qidirish",
  "select.language.title": "Tilni tanlang",
  "select.version.title": "Versiyani tanlang",
  "skip.link.title": "Tarkibga o'tish",
  "source.link.title": "Repozitoriyga o'tish",
  "source.file.date.updated": "Oxirgi yangilanish",
  "source.file.date.created": "Yaratildi",
  "tabs.title": "Yorliqlar",
  "toc.title": "Mundarija",
  "top.title": "Yuqoriga qaytish"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks for providing! Added in ec441cdab.

**Comment by squidfunk** [comment]
Released as part of 8.2.15.

**Comment by malikovss** [comment]
> Released as part of 8.2.15.

Great!
Thank you!
